packages/appstore/src/components/loader/cfd-toggle-skeleton/cfdToggleSkeleton.scss

```
@@ -0,0 +1,25 @@
+@import '../skeletonLoader.scss';
+
+.cfd-toggle-skeleton-loader {
+    display: none;
+    flex-direction: column;
+    width: 100%;
+    gap: 4px;
+    @include mobile {
+        display: flex;
```

I suggest if you want to only show on mobile only instead of `display: flex`, use `display: block`. Reason why `flex` has its own special stylings and `block` is just to show element there inside the page.
```suggestion
        display: block;
```